### ISSUE TITLE
Is this supposed to support for array?

### ISSUE THREAD

**Issue by joannechan** [cognitom/momy#13]
How should I represent the array field in the json file?

**Comment by cognitom** [comment]
Hi @joannechan, `momy` supports only [these types described in README](https://github.com/cognitom/momy#field-names-and-types).

We may have two options, I think.

1. make `momy` more customizable: *custom converter* or something
2. add [JSON data type](https://dev.mysql.com/doc/refman/5.7/en/json.html) support

The latter one would be easier to implement, but analyzing `JSON` type may have some difficulty. 

*Note*: MySQL has no `array` data type, but in v5.7 they introduced the new type for `JSON`.

**Comment by cognitom** [comment]
To achieve the first option, we may need to allow JavaScript style configuration (`momyfile.js`), instead of `momyfile.json`.

```javascript
// momyfile.js could be like this...?

const config = {
  "src": "mongodb://localhost:27017/dbname",
  "dist": "mysql://root:password@localhost:3306/dbname",
  "prefix": "t_",
  "case": "camel",
  "collections": {
    "collection1": {
      "_id": "number",
      "field1": "string",

      // This doesn't work at this point,
      // but this type of conversion could be useful in some cases.
      "field2": {
        "type": "string",
        "convert": val => Array.isArray(val) ? val.join('\t') : val
      }
    }
  }
}

export default config
```

**Comment by joannechan** [comment]
@cognitom Have this support for json datatype would be hugely beneficials.  Is this something that's going to be supported in the near future?  I am comparing this with mosql which only support postgres.

**Comment by Xsmael** [comment]
i strongly agree with @joannechan  supporting arrays would be so much useful especially when it comes to mongoDB, its hard to find a database without arrays. But is it possible to treat the array as a collection and documents inside would be the collection documents ? since they also have an _id field

**Comment by game5413** [comment]
Maybe my contribute progress #31  fit with your requirements, i had already implement feature that if some column have data type like array on json, it will be maping to new table and adding parent ID to that row, just add some relations object key on momyfile.json to make it read the mapping.

cc @joannechan , @Xsmael (sorry for directly mention, just want to be helpful)

PS: Sorry, if my english language so bad.
